cogs/profile.py

```diff
@@ -190,6 +190,16 @@ async def chunithm(
 
     @commands.hybrid_command(name="rename")
     async def rename(self, ctx: Context, *, new_name: str):
+        """Use magical power to change your IGN. Please note that this will change the actual display name of your CHUNITHM account.
```

Comment:
```suggestion
        """Use magical power to change your IGN. 
        
        Please note that this will change the actual display name of your CHUNITHM account.
```
The first line (the synopsis) should be shorter than 100 characters, due to Discord slash command limitations.